presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -339,7 +339,7 @@ protected static ResultSet getColumns(JdbcTableHandle tableHandle, DatabaseMetaD
     }
 
     @Override
-    public ConnectorSplitSource getSplits(JdbcIdentity identity, JdbcTableHandle tableHandle)
+    public ConnectorSplitSource getSplits(ConnectorSession session, JdbcIdentity identity, JdbcTableHandle tableHandle)
```

Comment:
Then you should remove `identity` parameter. it's part of the session

Reply:
Great :) One parameter less